docs/documentation/upgrading/topics/changes/changes-26_3_0.adoc

```diff
@@ -132,3 +132,11 @@ a new parameter `subGroupsCount` was introduced to the following endpoints:
 
 With this parameter, clients can decide whether the count should be returned to each individual group returned. To not break existing deployments,
 this parameter defaults to `true` so that the count is returned if the parameter is not set.
+
+=== Upgrade procedure changed for the distribution
+
+If you are upgrading {project_name} by downloading the distribution, the upgrade procedure has been changed. Previously it recommended copying over the contents from the `conf/` folder from the old to the new installation.
+The new procedure recommends to re-apply any changes to files in the `conf/` in case you modified any files in that directory, and leave them otherwise unchanged.
```

Comment:
Should we limit this only to `cache-ispn.conf` file? We don't have any defaults in `keycloak.conf` so copying should not be breaking.

Reply:
Added another commit, please let me know if this addresses your comment.